### ISSUE TITLE
empty() check on request triggers NoNonceVerification

### ISSUE THREAD

**Issue by Rarst** [WordPress-Coding-Standards/WordPress-Coding-Standards#1095]
```php
if ( ! empty( $_REQUEST['argument'] ) ) {
```

triggers `WordPress.CSRF.NonceVerification.NoNonceVerification`.

I feel `empty()` case should be under one of white lists, as effectively it is a cast to a guaranteed harmless boolean?

**Comment by JDGrimes** [comment]
It still indicates that a form submission, or some other request, that should possibly be accompanied by an nonce, is triggering some code to run (whatever code is in the `if` statement, in your example). So that code should possibly be protected by an nonce, hence the warning.

**Comment by Rarst** [comment]
I know the underlying theory... but every time I am hitting this in practice it's some small thing and it's just being sooo annoying. :)
